pyaedt/edb_core/edb_data/primitives_data.py

```diff
@@ -896,6 +894,60 @@ def __init__(self, app, arc):
         self._app = app
         self.arc_object = arc
 
+    @property
+    def start(self):
+        """Get the coordinates of the starting point.
+
+        Returns
+        -------
+        list
+            List containing the X and Y coordinates of the starting point.
+
+
+        Examples
+        -------
+        >>> appedb = Edb(fpath, edbversion="2023.2")
+        >>> start_coordinate = appedb.nets["V1P0_S0"].primitives[0].arcs[0].start
+        >>> print(start_coordinate)
+        [x_value, y_value]
+        """
+        point = self.arc_object.Start
+        return [point.X.ToDouble(), point.Y.ToDouble()]
+
+    @property
+    def end(self):
+        """Get the coordinates of the ending point.
+
+        Returns
+        -------
+        list
+            List containing the X and Y coordinates of the ending point.
+
+        Examples
+        -------
+        >>> appedb = Edb(fpath, edbversion="2023.2")
+        >>> end_coordinate = appedb.nets["V1P0_S0"].primitives[0].arcs[0].end
+        >>> print(end_coordinate)
+        [x_value, y_value]
+        """
+        point = self.arc_object.End
+        return [point.X.ToDouble(), point.Y.ToDouble()]
+
+    @property
+    def height(self):
+        """Get the height of the arc.
+
+        Returns
+        -------
+        float
+            Height of the arc.
+        Examples
```

Comment:
```suggestion

        Examples
```
Blank line needed before "Examples" section.